api/types/container/container_wait.go

```diff
@@ -15,7 +15,7 @@ type ContainerWaitOKBodyError struct {
 	Message string `json:"Message,omitempty"`
 }
 
-// ContainerWaitOKBody container wait o k body
+// ContainerWaitOKBody OK response to ContainerWait operation
```

Comment:
:+1: definitely looks better this way (was a bit worried about this change because of this)